### PR TITLE
Adds a test for deploying two bridged instances in the same network

### PR DESCRIPTION
**Pull request by ajsantander**
We may end up deleting the actual test, but it will pave the way for deployments of an instance in L2, and testing it's interaction with L1 instances.

#### Overview
* New test in `test/multi/index.js`. Run it with `npm run test:multi`.
* `publish/deploy` now looks into a `params.json` file for default parameter overrides. Will prompt the operator for confirmation on any overiides.

### CODE REVIEW

File: publish/deployed/local-ovm/rewards.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[
```
Comment (justinjmoses) [inline publish/deployed/local-ovm/rewards.json:1]: this can just be an empty `[]`

Comment (ajsantander) [reply publish/deployed/local-ovm/rewards.json:1]: I thought we needed to tell the ovm instance to have no pre-minted tokens?

Comment (justinjmoses) [reply publish/deployed/local-ovm/rewards.json:1]: no this is for LP staking rewards - not relevant here. It's used in regular local to test out the configuration of these but not relevant for L2-ovm

File: publish/src/commands/deploy.js [hunk 2]
```diff
@@ -30,21 +30,7 @@ const {
...
-		TARGET_THRESHOLD,
-		LIQUIDATION_DELAY,
-		LIQUIDATION_RATIO,
-		LIQUIDATION_PENALTY,
-		RATE_STALE_PERIOD,
-		EXCHANGE_FEE_RATES,
-		MINIMUM_STAKE_TIME,
-		AGGREGATOR_WARNING_FLAGS,
-	},
+	defaults,
```
Comment (justinjmoses) [inline publish/src/commands/deploy.js:33]: this needs to be loaded in right?

Comment (ajsantander) [reply publish/src/commands/deploy.js:33]: Loaded in?

I'm just not destructuring it as a way to enforce going through `getDeployParameter()`

File: publish/deployed/local-ovm/synths.json [hunk 3]
```diff
@@ -0,0 +1,284 @@
+[
+	{
+		"name": "sUSD",
+		"asset": "USD"
+	},
+	{
```
Comment (justinjmoses) [inline publish/deployed/local-ovm/synths.json:6]: only need sUSD

File: publish/deployed/local-ovm/config.json [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+	"SynthUtil": {
+		"deploy": true
+	},
+	"DappMaintenance": {
+		"deploy": true
+	},
+	"SynthsUSD": {
+		"deploy": true
+	},
+	"SynthsEUR": {
```
Comment (justinjmoses) [inline publish/deployed/local-ovm/config.json:98]: Can you remove all Synth*, TokenState* and Proxy* that isn't sUSD / Synthetix / FeePool

File: publish/deployed/local-ovm/feeds.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{
+	"SNX": {"asset": "SNX", "feed": "0x0"},
```
Comment (justinjmoses) [inline publish/deployed/local-ovm/feeds.json:2]: only need SNX

File: publish/src/commands/deploy.js (hunk 2 above)
Comment (justinjmoses) [reply publish/src/commands/deploy.js:33]: Sorry i misread this as index.js. Please ignore.

File: publish/deployed/local-ovm/rewards.json (hunk 1 above)
Comment (ajsantander) [reply publish/deployed/local-ovm/rewards.json:1]: Ah sorry, I miss-understood. Ok.

**Pull request merged**